src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -323,7 +339,7 @@ public async Task<string> GetSyncTriggersPayload()
 
         internal async Task<IEnumerable<JObject>> GetFunctionTriggers(IEnumerable<FunctionMetadata> functionsMetadata, ScriptJobHostOptions hostOptions)
         {
-            var durableTaskConfig = await ReadDurableTaskConfig();
+            GlobalDurableConfig durableTaskConfig = await ReadDurableTaskConfig();
```

Comment:
Can we do this lazily so we only do all the file ops if there are actually Durable functions? That way we aren't performing a bunch of unecessary work for people not using Durable

Reply:
Good idea.

Reply:
I now check for the presence of any durable triggers before I look for the config.